css/style.css

```diff
@@ -0,0 +1,73 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: "oppins", sans-serif;
+  background-image: url("../img/background.png");
+  background-repeat: no-repeat;
+}
```

Comment:
- [ ] I noticed that the `background-image` you used for your design is different from the one in Figma. Kindly use the correct `background-image` and position it correctly. It is important you use the `background-image` correctly so that your design looks familiar with that of the Figma template that you chose. Kindly see the difference between the two snapshots I attached 😎 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/221054680-b9ee2b60-3e2b-4a49-8378-39b347c44eab.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/221054723-e23ef97a-603f-42f4-8d32-05dc3deb2857.png)